### ISSUE TITLE
Can you add a song to Spotify `Favorites` from the workflow?

### ISSUE THREAD

**Issue by nikitavoloboev** [vdesabou/alfred-spotify-mini-player#261]
Basically triggering this heart action on a song that's currently playing:

![2018-10-10 at 11 55](https://user-images.githubusercontent.com/6391776/46728583-6d387a00-cc83-11e8-8ac4-838e2f9a93e8.png)

**Comment by lukesmurray** [comment]
Seems like this is supported by some other players such as [spotify-tui](https://github.com/Rigellute/spotify-tui) so it seems possible this has been added to the api.

**Comment by vdesabou** [comment]
Yes I think that favorite a track is just the equivalent of adding it to your music, see https://alfred-spotify-mini-player.com/articles/alfred-playlist/

so it is already possible actually
